src/utils/keyManagement.js

```diff
@@ -25,21 +25,17 @@ export const eraseHpAdminKeypair = () => {
 // Use singleton pattern
 // Return null when trying to initialize with no params
 // TODO: the default email and password here are for testing only and should be removed before merging
-export const getHpAdminKeypair = async (email = "test@test.com", password = "passw0rd") => {
+export const getHpAdminKeypair = async (email, password) => {
```

Comment:
This is still called on line 43 below with 0 args

Reply:
But won't you get an error for calling a function with wrong number of args? Or does it default to `undefined`?

Reply:
it defaults to `undefined`